src/DynamoCoreWpf/Views/GuidedTour/RatingControl.xaml.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Controls.Primitives;
+
+namespace Dynamo.Wpf.Views.GuidedTour
+{
+    /// <summary>
+    /// Interaction logic for RatingControl.xaml
+    /// </summary>
+    public partial class RatingControl : UserControl
+    {
+        public RatingControl()
+        {
+            InitializeComponent();
+        }
+
+        public static readonly DependencyProperty ValueProperty = DependencyProperty.Register("RatingControl", typeof(int), typeof(RatingControl),
+          new FrameworkPropertyMetadata(0, FrameworkPropertyMetadataOptions.BindsTwoWayByDefault, new PropertyChangedCallback(RatingChanged)));
```

Comment:
Can we define this in xaml above?

Reply:
@QilongTang I was not able to find a way to define this in the xaml, I think only can be defined in the code behind, or do you know how to register a dependency property in xaml?

Reply:
I think it's unclear why this *needs* to be a dependency property?

Reply:
I guess it should actually be called `"Value"` ?

Reply:
@mjkkirschner  you were right, the name was changed to "Value" in the next commit:  https://github.com/DynamoDS/Dynamo/pull/11834/commits/660ad2892deb9b86c70463b7949bc26de518b5f9
Thanks